task/01-strings-tasks.js

```diff
@@ -237,7 +243,7 @@ function encodeToRot13(str) {
  *   isString(new String('test')) => true
  */
 function isString(value) {
-  throw new Error('Not implemented');
+  return Object.prototype.toString.call(value) === '[object String]'; 
```

Comment:
It's now so obvious to compare such a value.

I believe in this case we know that a string could be a primitive or an object, so it should be enough:
`return typeof value === 'string' || value instanceof String`

Reply:
agree, changed it